tests/random_attack.py

```diff
@@ -292,6 +290,19 @@ def set_key_columns(self, frame):
                                     "    DT.key = %r\n\n"
                                     % names)
 
+    def delete_columns_array(self, frame):
+        # datatable supports a shape of n x 0 for non-zero n's, while
+        # python doesn't, so we never remove all the columns from a Frame.
+        if frame.ncols < 2:
+            return
+        s = self.random_array(frame.ncols - 1, positive=True)
+        s = sorted(set(s))
```

Comment:
Theoretically, you don't have to either sort or deduplicate the list of columns that you want to remove.

Reply:
Yeah, left it by accident editing `delete_rows_array()`.